public/app/features/canvas/element.ts

```diff
@@ -64,3 +65,4 @@ export interface CanvasElementItem<TConfig = any, TData = any> extends RegistryI
 
 export const defaultBgColor = '#D9D9D9';
 export const defaultTextColor = '#000000';
```

Comment:
This change looks good. I'm guessing for the two other places that we use the `defaultTextColor` that we will want to always default to black text (in both rectangle + metric value) given the default of a background color for each of those elements

Reply:
Yes, I think `defautTextColor` works with the default background (which looks good for both themes) for rectangle and mv.